po/picard.pot

```diff
@@ -8,14 +8,14 @@ msgid ""
 msgstr ""
 "Project-Id-Version: picard 2.7.0.dev3\n"
 "Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
-"POT-Creation-Date: 2021-06-29 11:58+0200\n"
+"POT-Creation-Date: 2021-06-30 20:54-0300\n"
```

Comment:
Please do not regenerate `picard.pot` within the PR (we'll do it, but once the PR will be merged)

Reply:
OK o>